src/pineko/scale_variations.py

```diff
@@ -200,6 +201,8 @@ def compute_ren_sv_grid(grid_path, max_as, nf, target_path=None):
     if sv_as:
         rich.print(f"[green]Renormalization scale variations are already in the grid")
         return
+    # With respect to the usual convention here max_as is max_as-1
+    max_as = max_as - 1
```

Comment:
Why?

Reply:
As it is written in the comment (`# With respect to the usual convention here max_as is max_as-1`), we are usually adopting the convention such that `max_as = 1` is LO, `max_as = 2` is NLO and so on (in particular the pineappl function `order_mask` is using this). This means that, for instance, the check `check.contains_ren`, that I am doing just before this line,  is built taking this into account. However, all the functions that I have implemented for automatic scale variations are implemented with a different convention and they work decreasing the max_as by 1. If you prefer I can use another parameter instead of changing `max_as`.

Reply:
> However, all the functions that I have implemented for automatic scale variations are implemented with a different convention and they work decreasing the max_as by 1. If you prefer I can use another parameter instead of changing `max_as`.

But is this something local, is there a reason not to use the same `max_as` also in the functions you wrote?

Reply:
Of course I could adapt my functions to use the "other" `max_as` but I think that they are more clear as they are, which is the reason why I just changed the max_as value with which I call them. So yes, this is just something local

Reply:
While I may understand that is clearer as you wrote, it might still be easier to have a single definition of `max_as` around (if possible consistent among all programs, but at least consistent in Pineko).

If we read that code, and there is `max_as` around, but we forget to read first the translation, it will make everything harder to understand.
It is not a bad choice on its own, but consistency is good for maintenance.

Reply:
I'd just like to understand to which level clearness would be compromised. If it is a "not that much", I'd rather use the original definition of `max_as`

Reply:
Well basically now all the functions depend on `max_as`. Using instead the other definition all the functions will depend on `max_as - 1`. We will have then a couple of functions in which every time `max_as` is used we need to put `max_as-1`. So at the end instead of having only one place in which we do `max_as - 1` we will have three or four in a couple of functions (that however are not "top-level" functions) .